pkg/apis/acme/v1alpha3/types_issuer.go

```diff
@@ -81,14 +102,25 @@ const (
 	HS512 HMACKeyAlgorithm = "HS512"
 )
 
+// Configures an issuer to solve challenges using the specified options.
+// Only one of HTTP01 or DNS01 may be provided.
```

Comment:
```suggestion
// ACMEChallengeSolver configures an issuer to solve challenges using the specified options.
// Only one of HTTP01 or DNS01 may be provided.
```

Reply:
I don't think this is a worthwhile change - these docs are end-user facing, and `ACMEChallengeSolver` isn't really something they're going to be familiar with